### ISSUE TITLE
Does Asset Rev work with Craft 3.5?

### ISSUE THREAD

**Issue by anthonator** [clubstudioltd/craft-asset-rev#28]
We recently upgraded to Craft 3.5 and Asset Rev is now disabled and won't re-enable through the dashboard. I'm curious if anybody has gotten Asset Rev 6.0.2 to work with Craft 3.5 (specifically 3.5.6).

**Comment by scottwakefield** [comment]
Hi @anthonator,

I've just upgraded an existing site that uses Asset Rev 6.0.2 to Craft 3.5.6 and everything looks fine.

![image](https://user-images.githubusercontent.com/4173571/92233259-dba20900-eea7-11ea-817a-f1b795857782.png)

![image](https://user-images.githubusercontent.com/4173571/92233359-0c823e00-eea8-11ea-8ddc-17017144dfd2.png)

I've also tried disabling the plugin and then enabling it again and that worked, too.

Are you still having trouble? If so, are you using Project Config or anything else that might control whether a plugin is enabled or not?
